packages/polaris-viz/src/components/TooltipWrapper/utilities/getAlteredLineChartPosition.ts

```diff
@@ -42,7 +28,7 @@ export function getAlteredLineChartPosition(
   //
 
   if (props.isPerformanceImpacted) {
-    y = chartBounds.y - (scrollContainer?.scrollTop ?? 0) ?? 0;
+    y = chartBounds.y - (scrollContainer?.scrollTop ?? 0);
```

Comment:
It was an extra check that never really did anything becuase `chartBounds.y - (scrollContainer?.scrollTop ?? 0)` would never be `null`. So that extra `?? 0` wasn't doing anything constructive.